### DISCUSSION TITLE
Have some of the integration pieces been disabled?  Cannot get the Error state

### DISCUSSION THREAD

**Discussion by MrNickIE** [Thomas55555/husqvarna_automower#416] [Q&A]
Hello, we are wanting to have an automation within the house to alert us on screens if the mower is in error state.  However it appears those diagnostic sensors are now disabled?

![image](https://user-images.githubusercontent.com/82377682/230711121-d15b5efb-2aa1-43b0-bb17-b019b74fc1b9.png)

**Comment by Thomas55555** [comment]
Yes, they are disabled. Just enable them.

**Reply by MrNickIE** [reply]
Thanks.  Not sure if I disabled them?  Either way I am enabling now. Just did r want to break something. Thanks!
